packages/theme-chalk/src/message-box.scss

```diff
@@ -78,10 +80,8 @@
     border: none;
     outline: none;
     background: transparent;
-    font-size: var(
-      #{getCssVarName('message-close-size')},
-      map.get($message, 'close-size')
-    );
+    font-size: var(#{getCssVarName('message-close-size')},
+      map.get($message, 'close-size'));
```

Comment:
@921 I suggest that the code here and below should be kept in the original format to avoid unnecessary modifications.